### PR TITLE
Hybrid functional families

### PR DESCRIPTION
**Pull request by BKaperick**
Copy-pasted some code for `:gga` family to apply to `:hyb_gga` - needed for https://github.com/JuliaMolSim/DFTK.jl/pull/717.

Also fix typo in `exx_coefficient` printing.

### CODE REVIEW

File: src/evaluate.jl [hunk 1]
```diff
@@ -2,13 +2,21 @@
 # :input refers to input argument and numbers to the derivative order
 const ARGUMENTS = Dict(
     :lda => [[:zk], [:vrho], [:v2rho2], [:v3rho3], [:v4rho4], ],
+    :hyb_lda => [[:zk], [:vrho], [:v2rho2], [:v3rho3], [:v4rho4], ],
```
Comment (mfherbst) [inline src/evaluate.jl:5]: I don't like this repetition. Perhaps you can just do in a second step (i.e. below the definition of the constant):
```ARGUMENTS[:hyb_lda] = ARGUMENTS[:lda]``` and similarly for the other things.

File: test/runtests.jl [hunk 2]
```diff
@@ -47,6 +47,22 @@ end
...
 
+@testset "Hybrid LDA and Hybrid GGA evaluate" begin
+    rho = [0.1, 0.2, 0.3, 0.4, 0.5]
+    sigma = [0.2, 0.3, 0.4, 0.5, 0.6]
+    result = zeros(Float64, 5)
+
+    # LDA
+    hyb_lda_xc = Functional(:hyb_lda_xc_bn05)
+    result = evaluate(hyb_lda_xc, rho=rho, derivatives=0).zk
+    @test result ≈ [-0.162837, -0.234719, -0.286980, -0.329442, -0.365816] atol=1e-5
```
Comment (mfherbst) [inline test/runtests.jl:58]: I trust you computed these values in some other way (i.e. not using Libxc.jl). Is this correct?

**Review by mfherbst — COMMENTED** [review]
LGTM except the one open question.

File: test/runtests.jl (hunk 2 above)
Comment (BKaperick) [reply test/runtests.jl:58]: Yep :-) I just did it with libxc in C with their example script on the website [here](https://tddft.org/programs/libxc/manual/libxc-5.1.x/), initializing [func_id to 588](https://gitlab.com/libxc/libxc/-/blob/master/src/funcs_key.c) for `hyb_lda_xc_bn05` and 496 for `hyb_gga_x_s12h`, and adding `case XC_FAMILY_HYB_LDA:` in the switch statement.  (I can send you the script if that's not clear).  The results are the same up to 1e-5.

Comment (mfherbst) [reply test/runtests.jl:58]: great. Then I'll merge!

**Pull request merged**